server/controller/src/main/java/ai/starwhale/mlops/api/UserApi.java

```diff
@@ -91,9 +101,24 @@ ResponseEntity<ResponseMessage<PageInfo<UserVO>>> listUser(
                     schema = @Schema(implementation = UserVO.class)))
         })
     @GetMapping(value = "/user/current")
+    @PreAuthorize("hasAnyRole('OWNER', 'MAINTAINER', 'GUEST')")
     ResponseEntity<ResponseMessage<UserVO>> getCurrentUser();
 
 
+    @Operation(summary = "Check Current User password")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "ok")})
+    @PostMapping(value = "/user/current/pwd")
+    @PreAuthorize("hasAnyRole('OWNER', 'MAINTAINER', 'GUEST')")
+    ResponseEntity<ResponseMessage<String>> checkCurrentUserPassword(@RequestBody UserCheckPasswordRequest userCheckPasswordRequest);
+
+
+    @Operation(summary = "Update Current User password")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "ok")})
+    @PutMapping(value = "/user/current/pwd")
```

Comment:
Will it conflict with the API which uses `userId` as the URI part if the username is `current`?

Reply:
Sorry, I forgot `userId` is a number